core/viewer-wicket/ui/src/main/java/org/apache/isis/viewer/wicket/ui/pages/entity/EntityPage.java

```diff
@@ -100,29 +99,22 @@ private EntityPage(final PageParameters pageParameters, final EntityModel entity
         this(pageParameters, entityModel, null);
     }
 
-    public EntityPage(IsisWebAppCommonContext commonContext, ManagedObject adapter) {
```

Comment:
was this an unused constructor?

Reply:
Note: this removed constructor has same signature as the one that we got after refactoring below.